doc/user/sql/create-sink.md

```diff
@@ -29,19 +29,17 @@ _item&lowbar;name_ | The name of the source or view you want to send to the sink
 **KAFKA BROKER** _host_ | The Kafka broker's host name.
 **TOPIC** _topic_ | The Kafka topic to write to.
 **CONFLUENT SCHEMA REGISTRY** _url_ | The URL of the Confluent schema registry to get schema information from.
-**SCHEMA FILE** _path_ | The absolute path to a file containing the schema.
-**SCHEMA** _inline&lowbar;schema_ | A string representing the schema.
+**FILE** _topic_ | The absolute path of the file you want to use as the root of the sink. 
```

Comment:
not sure how to best convey that these are not the actual file or topic name but rather file / topic name prefixes or "virtual" names